crates/matrix-sdk-crypto/src/types/events/olm_v1.rs

```diff
@@ -182,6 +185,52 @@ where
     pub content: C,
 }
 
+impl<C: EventType + Debug + Sized + Serialize> Serialize for DecryptedOlmV1Event<C> {
```

Comment:
A comment as to why this is needed/what it does differently from `derive(Serialize)` might be helpful.